plugins/vue-filters.js

```diff
@@ -28,3 +30,25 @@ Vue.filter('optimisedImageUrl', (imageUrl, contentType) => {
   }
   return imageUrl;
 });
+
+Vue.filter('inCurrentLanguage', (val, locale) => {
+  let languageKeys = ['en', 'eng', 'def', 'und'];
+  const currentLocale = locales.find(l => l.code === locale);
+
+  if (currentLocale.code !== 'en' || currentLocale.isoAlpha3 !== 'eng') {
+    languageKeys.unshift(currentLocale.code);
+    languageKeys.unshift(currentLocale.isoAlpha3);
+  }
+
+  for (let key of languageKeys) {
```

Comment:
Could you add a comment explaining what all of this is doing as I can imagine it being tricky to scan when it's not fresh in our minds.

Reply:
I have added some comments to the filter.